src/main/groovy/org/pih/warehouse/outbound/ImportPackingListItem.groovy

```diff
@@ -65,6 +67,8 @@ class ImportPackingListItem implements Validateable {
         if (!person && source['recipient']) {
             // We want to indicate if a recipient was not found, but the search term was given
             obj.recipientFound = false
+            // returns a dummy person object to add more context in a response of what person was not found
+            return new Person(firstName: source['recipient'], lastName: "")
```

Comment:
I can't see it being included in the `recipient` validator in the same way, as you included it in the bin location validator.